.github/labeler.yml

```diff
@@ -1,25 +1,25 @@
 # https://github.com/actions/labeler#common-examples
-# Adapted from https://github.com/rapidsai/cudf/blob/main/.github/CODEOWNERS
+# Adapted from https://github.com/rapidsai/cuxfilter/blob/main/.github/CODEOWNERS
 # Labels culled from https://github.com/rapidsai/cuxfilter/labels
 
 Python:
- - python/**
- - notebooks/**
+ - 'python/**'
+ - 'notebooks/**'
 
 libcuxfilter:
- - cpp/**
+ - 'cpp/**'
 
 CMake:
- - **/CMakeLists.txt
- - **/cmake/**
+ - '**/CMakeLists.txt'
+ - '**/cmake/**'
 
 Java:
- - java/**
+ - 'java/**'
```

Comment:
This section should match this repo's labels and the contents of the .github/CODEOWNERS file as a starting guide - PICs in this repo would then add or suggest more labels as needed